test/cmd/wait.sh

```diff
@@ -26,7 +26,14 @@ run_wait_tests() {
 
     create_and_use_new_namespace
 
-    ### Wait for deletion using --all flag
+    # wait --for=create should time out
+    set +o errexit
+    # Command: Wait with jsonpath support fields not exist in the first place
+    output_message=$(kubectl wait --for=create deploy/test-1 --timeout=1s 2>&1)
```

Comment:
Would you prefer adding successful execution tests in followup PRs ?. This is covered in unit tests but I think, it is better to have a at least 1 successful execution in integration as well.

Reply:
Will definitely add one.